sqlfluff_templater_dataform/templater.py

```diff
@@ -88,8 +90,15 @@ def has_js_block(self, sql: str) -> bool:
         return bool(pattern.search(sql))
 
     def replace_blocks(self, in_str: str) -> str:
-        pattern = re.compile(CONFIG_BLOCK_PATTERN, re.DOTALL)
-        return re.sub(pattern, '', in_str)
+        for block_pattern in [
+            CONFIG_BLOCK_PATTERN,
+            PRE_OPERATION_BLOCK_PATTERN,
+            POST_OPERATION_BLOCK_PATTERN
+        ]:
```

Comment:
You should add patterns to line 128's patterns.
These are needed to keep track of the number of rows in the lint.